features.yaml

```diff
@@ -1,258 +1,282 @@
   - name: "TLS termination and SNI Support in Gateways"
     level:
-     - checklist:
-       maturity: Stable
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Stable
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "SNI (multiple certs) at ingress"
     level:
-     - checklist:
-       maturity: Stable
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Stable
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "Locality load balancing"
+    link: "/docs/tasks/traffic-management/locality-load-balancing/"
     level:
-     - checklist:
-       maturity: Beta
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Beta
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "Enabling custom filters in Envoy"
     level:
-     - checklist:
-       maturity: Alpha
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Alpha
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "CNI container interface"
     level:
-     - checklist: features/cni.md
-       maturity: Alpha
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: features/cni.md
+      maturity: Alpha
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "Sidecar API"
+    link: "/docs/reference/config/networking/sidecar/"
     level:
-     - checklist:
-       maturity: Beta
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Beta
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "DNS Proxying"
+    link: "/docs/ops/configuration/traffic-management/dns-proxy/"
     level:
-     - checklist:
-       maturity: Alpha
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Alpha
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "Service APIs"
     level:
-     - checklist: features/service_apis.md
-       maturity: Alpha
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: features/service_apis.md
+      maturity: Alpha
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "Kubernetes Service APIs"
+    link: "/docs/tasks/traffic-management/ingress/gateway-api/"
     level:
-     - checklist:
-       maturity: Alpha
-       nextExpectedPromotion:
-       area: Traffic Management
+      checklist: ""
+      maturity: Alpha
+      nextExpectedPromotion: ""
+    area:  Traffic Management
   - name: "Prometheus Integration"
+    link: "/docs/tasks/observability/metrics/querying-metrics/"
     level:
-     - checklist:
-       maturity: Stable
-       nextExpectedPromotion:
-       area: Observability
-  - name: "Client and Server Telemetry Reporting"
```

Comment:
This feature was removed with mixer, but we still list it, just pointing at the 1.6 docs. Removing it.